package.json

```diff
@@ -43,7 +43,7 @@
     "depcheck": "depcheck ."
   },
   "dependencies": {
-    "@ethersphere/bee-js": "^6.4.0",
+    "@ethersphere/bee-js": "^6.4.1-pre.1",
```

Comment:
can you point me to "what's new" in pre.1 ?

Reply:
the tests should work in this test environment

Reply:
Can you please update to `6.6.0`?